src/recaptcha-manager.ts

```diff
@@ -15,20 +20,34 @@ export interface RecaptchaManagerInterface {
 }
 
 export class RecaptchaManager implements RecaptchaManagerInterface {
+  private static readonly DEFAULT_TIMEOUT = 10000;
+
   private lazyLoader: LazyLoaderServiceInterface;
 
   private defaultSiteKey?: string;
 
   private recaptchaCache: Record<string, RecaptchaWidget> = {};
 
+  /**
+   * How long in milliseconds to wait for the recaptcha library to load before timing out.
+   */
+  private timeout: number;
```

Comment:
q: should this value start with the DEFAULT_TIMEOUT?

Reply:
Sure, couldn't hurt!